rotkehlchen/tests/api/test_data_import.py

```diff
@@ -88,7 +89,7 @@ def test_data_import_cointracking(rotkehlchen_api_server, file_upload):
     result = assert_proper_sync_response_with_result(response)
     assert result is True
     # And also assert data was imported successfully
-    assert_cointracking_import_results(rotki)
+    assert_cointracking_import_results(rotki, websocket_connection)
```

Comment:
nitpick: But here you can leave as it's test and obvious. If more than 1 argument, use kwargs

```suggestion
    assert_cointracking_import_results(rotki, websocket_connection)
```